templates/visuals/.api/v1.7.0/schema.capabilities.json

```diff
@@ -820,6 +824,92 @@
                         }
                     }
                 },
+                "fillRule" : {
```

Comment:
is this auto-generated in the settings generator as well?

Reply:
Not sure. I am not familiar about settings generator yet.

Reply:
I will figure out